04-onion-routing.md

```diff
@@ -599,6 +589,48 @@ set `payment_constraints.max_cltv_expiry` to restrict the lifetime of a blinded
 route and reduce the risk that an intermediate node updates its fees and rejects
 payments (which could be used to unblind nodes inside the route).
 
+### Inside `encrypted_recipient_data`: `encrypted_data_tlv`
+
+The `encrypted_data` is a TLV stream, encrypted for a given blinded node, that
```

Comment:
Use `encrypted_recipient_data` here as well, for consistency.

Reply:
Replaced everywhere, thanks!